lynx/common/player.py

```diff
@@ -10,10 +13,68 @@ class Player(Serializable):
     player_resources: Dict[str, int] = field(default_factory=dict)
     drop_area: Vector = field(default_factory=Vector)
     agents: dict[str, Agent] = field(default_factory=dict)
+    max_agents: int = 1
+
+    BASE_COSTS = {
+        "Wood": 6,
+        "Stone": 6,
+    }
+
+    @property
+    def price_multiplier(self) -> int:
```

Comment:
What does this function do? It multiplies a price. So:
```suggestion
    def multiply_price(self) -> int:
```

Reply:
Property nazwyamy rzeczownikami - to jest pole NIE metoda

Reply:
My bad, I didn't notice